db/migrate/20170702021929_create_law_offices.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLawOffices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :law_offices do |t|
+      t.string :law_office
```

Comment:
How do you feel about changing this column to be called `name`?  It feels more natural to me to say `law_office.name` instead of `law_office.law_office`.